### ISSUE TITLE
Resources side menu keyboard navigation issues

### ISSUE THREAD

**Issue by nellihol** [ecoacoustics/website#48]
When navigating the resources page using only the tab key the focus goes from search to first item on the side menu to breadcrumb. 

For better accessibility for keyboard and screen reader users the navigation order should be logical.

- [ ] Add a hidden link to skip the navigation for screen reader users
- [ ] the tab key should go through all the side navigation elements in a logical order

I am suspecting this is related to the shadow DOM that Shoelace uses.
